File: tests/human/x/java/MapNestedAssign.java
import java.util.*;
public class MapNestedAssign {
    public static void main(String[] args) {
        Map<String, Map<String, Integer>> data = new HashMap<>();
        Map<String, Integer> inner = new HashMap<>();
        inner.put("inner", 1);
        data.put("outer", inner);
        data.get("outer").put("inner", 2);
        System.out.println(data.get("outer").get("inner"));
    }
}

File: tests/human/x/scala/map_nested_assign.scala
object map_nested_assign {
  def main(args: Array[String]): Unit = {
    val data = scala.collection.mutable.Map("outer" -> scala.collection.mutable.Map("inner" -> 1))
    data("outer")("inner") = 2
    println(data("outer")("inner"))
  }
}
